### ISSUE TITLE
[Discussion] [Classic] All hunter pets have the same stats

### ISSUE THREAD

**Issue by bdebaere** [cmangos/issues#1679]
All hunter pets get their stats from the information in `SELECT * FROM pet_levelstats WHERE creature_entry = 1`. No matter if it is a boar, tiger, turtle or raptor. Increasing these stats in this table, except for hp and armor, have no effect whatsoever as far as I can tell.

Are we certain this is correct? What's the use of taming one pet over the other if not for stat differences? Why doesn't increasing the stamina in pet_levelstats increase the health?

**Comment by cala** [comment]
@Phatcat did work on that topic last year (or the previous one, time flies by so quickly). I remember we had a discussion on pet stats initialisation before he released it in one of his patch.
It was not perfect but still a nice improvement on the previous state and IIRC work was stop at the point of (re)implementing some stats initialisation spells for each kind of pet.

Since, pet system had a many changes, for good or ill, and I don't know if the current design is still what he had in mind.

**Comment by Tobschinski** [comment]
Didn't check if they are used or supposed to be used, but there's a bunch of `Tamed Pet Passive (DND)` spells modifying damage, health and armor. These are most likely the stat differences you were looking for.

**Comment by killerwife** [comment]
Yes, and they are automatically applied from DBC. But from my past experience with DBCs, I would not be surprised if they had mistakes. They are applied based on the type of pet. Generic ones are given to all (for example there is a generic one that handles orc racial passive), and particular ones to for example Crabs.

**Comment by Phatcat** [comment]
What tobs said, hunter pets all have the same stats and are modified through passives.

**Comment by Phatcat** [comment]
Cala I have nothing in mind for this project anymore.

**Comment by Phatcat** [comment]
I guess since noone cares to dig it out from my pet patches or remember, here is the proof; http://web.archive.org/web/20070222092151/http://petopia.brashendeavors.net/html/articles/stats_families.shtml

And here is me getting schooled on how stat modifiers are applied to pets after I initially tried to hardcode it in; https://github.com/cmangos/mangos-classic/commit/86cb54fd6938d4b9045d65f259bcfa4edc5f59a7#commitcomment-17746378

Now, that was a fun trip down memory lane, but this issue ticket is invalid and can be closed, thank you.

**Comment by Phatcat** [comment]
To further prove my point here are 3 screenshots of different pets, 2 complete different species of pets all in level 60 with the same exact base stats; http://web.archive.org/web/20061201234625/http://tkasomething.com:80/attackspeeds.php

You may notice the last pet, the wolf, has more armor, but both the cats have a higher dps, which, is their racial modifiers shown as shown in my previous comment here; http://web.archive.org/web/20070222092151/http://petopia.brashendeavors.net/html/articles/stats_families.shtml - but they they all still have the same exact base stats. 136 strength, 100 agility, etc..

**Comment by Phatcat** [comment]
Last comment on this one talking about them supposed to have the same base stats; 

> All pets, no matter what their family, have the same base stats at each level. (There are a few rare exceptions that have inferior attributes. We call these "caster" pets, and discuss them separately.) Sometimes you can easily see these stats on the pet's UI panel -- for instance, with strength or armor -- and sometimes you have to figure out the base stat by working backwards from the final stats -- as is the case with health. Since these base values are the same across all pets, they don't play any role in helping you choose a pet. But sometimes knowing the base values, especially for health, can be useful. So here's a handy table! 

http://web.archive.org/web/20070224130459/http://petopia.brashendeavors.net/html/articles/stats_attributes.shtml

We are however not applying pet family modifier auras correctly (perhaps not at all), so atm there is absolutely 0 stat differences which is obviously wrong.

**Comment by killerwife** [comment]
I need an example of two pets that are meant to be different in vanilla.
